### PR TITLE
{bio,lang}[GCCcore/6.4.0,intel/2018a] Scrappie v1.3.2, CUnit v2.1-3

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node2490.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2b584399db38f747923458d4c26448dc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
gligar01.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c10e67c3a300e9bec0d6b6759649768c for a full test report.

File: easybuild/easyconfigs/s/Scrappie/Scrappie-1.3.2-intel-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    '47d08589bb9911d8572c70479760b4096018f3c32c9e63f71ce05e157f7fe7a3',  # Scrappie-1.3.2_fix-cblas-include.patch
+]
+
+builddependencies = [
+    ('CMake', '3.10.2'),
+    ('CUnit', '2.1-3'),
+]
+dependencies = [('HDF5', '1.10.1')]
+
+prebuildopts = 'sed -i "s/ -lblas / $LIBBLAS /g" CMakeFiles/scrappie.dir/link.txt && '
```
Comment (damianam) [inline easybuild/easyconfigs/s/Scrappie/Scrappie-1.3.2-intel-2018a.eb]: Define `sed -i "s/ -lblas / $LIBBLAS /g"` as a variable and use it in both `prebuildopts` lines?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2476.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2e5f1e8fe43698d35b347b96854d1db3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2007.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/55cd105f0f85e6fa98141e88624ea384 for a full test report.

**Pull request merged**